en/developer_guide/routing.rst

```diff
@@ -0,0 +1,110 @@
+Routing
+=======
+
+Base URL
+--------
+
+In Open Orchestra, the base url of a website is composed by the domain and eventually a language prefix.
+To manage all possibilities, Open Orchestra associate each website in the database with website aliases which contain:
+
+* **domain**: domain name
+* **language prefix**: used by Open Orchestra to define the language
+* **scheme** : to choose with which protocol page can be accessed
+
+Here is an example of some base URL generated for a website associate to two domains: ``dev.example.com`` and ``test.example.com``.
+Each of them, with english as default language and french as secondary one.
+
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| Alias Id | Scheme | Domain           | Language | Language prefix | Base URL                    |
++==========+========+==================+==========+=================+=============================+
+| 0        | http   | dev.example.com  | fr       | fr              | http://dev.example.com/fr   |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 1        | http   | dev.example.com  | en       | none            | http://dev.example.com      |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 2        | http   | test.example.com | fr       | fr              | http://test.example.com/fr  |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 3        | https  | test.example.com | en       | none            | https://test.example.com    |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+
+
+URL Pattern
+-----------
+
+The url pattern is added to the ``base URL`` to generate the url.
+For node, the url pattern can be choose by contributor or it is generated automatically from the page title.
+
+Url Pattern use Symfony rules :
+
+* Variables to exctract between {}, ex : `/exemple/{param}/test`
+* Pattern have to be unique
+
+See ``Url pattern`` chapter from `Node Configuration`_ for more details.
+
+Redirection
+-----------
+
+Open Orchestra allows user to determine redirection in the back office.
```

Comment:
"allows a user" or "allows users"
"to determine redirection" is not clear to me. Shouldn't we explain briefly the functionnal behavior ?